src/robot-trajectory-manager.ts

```diff
@@ -38,6 +38,7 @@ export interface Trajectory {
   segments: RawKnot[];
   robot_name: string;
   fleet_name: string;
+  map_name: string;
```

Comment:
Just to check is the `map_name` === Level name, right? 
Also if the `map_name` property is required you may want to add it to `trajectories.json` too. For the 
`npm run start:mock` to work

Reply:
add map_name to json 6910e26